src/plugins/dataflow-tool/microbit/hub.md

```diff
@@ -6,9 +6,9 @@ Downloadable link: https://makecode.microbit.org/_YpoCfzeTwCWP
 
 #### Outgoing strings to write,
 e.g. : `sat20.31`
-- `s`: "sensor"
+- `s`: "signal"
 - `a`: "hub A"
-- `t`: "temperature reading"
+- `t`: "temperature reading" (or `h` for humidity, or `r` for relays state),
```

Comment:
I don't understand this file at all, so maybe this is a dumb question. But why not make `h` and `r` additional lines here instead of including them all on the `t` line?

Reply:
Thanks - it was notes I didn't write very clearly.  I've tried to improve them. below.